src/main/java/reposense/report/RepoCloner.java

```diff
@@ -269,6 +273,7 @@ private boolean spawnPartialCloneProcess(RepoConfiguration config, boolean shoul
     /**
      * Spawns a process to create shallow partial clone of repo specified in {@code config}.
      * Waits for process to finish executing.
+     * For test environments, cloning is skipped if it has been done before and {@code shouldFreshClone} is false.
      */
     private boolean spawnShallowPartialCloneProcess(RepoConfiguration config, boolean shouldFreshClone) {
         assert(crp == null);
```

Comment:
The javadoc for `waitForCloneProcess` (in line 313) should include the `config` variable in the javadoc.